decentraland/unity-renderer
# refactor: new lib functions, moving code around

## What does this PR change?

Nothing, some minor functions extracted.

## How to test the changes?

1. Launch the explorer
2. Login as guest, or with an account
3. Ensure the world renders
